### PR TITLE
Empty out and deprecate astroquery.ibe module

### PR DESCRIPTION
**Pull request by lpsinger**
IRSA no longer supports the IBE interface. To access IRSA datasets programatically, users encouraged to use the [TAP protocol](http://irsa.ipac.caltech.edu/docs/program_interface/TAP.html) via [PyVO](http://pyvo.readthedocs.io/en/latest/dal/index.html#table-access-protocol).

### CODE REVIEW

**Comment by bsipocz** [comment]
@lpsinger @keflavich - I wonder whether it would make any sense to try to use `astroquery.utils.tap` to keep this service in astroquery? I guess that would though need some non trivial amount of work so we better treat as adding a new service (so probably it doesn't end up at the top of the queue).

**Comment by lpsinger** [comment]
> @lpsinger @keflavich - I wonder whether it would make any sense to try to use astroquery.utils.tap to keep this service in astroquery?

Actually, that's a very good idea! I didn't know that module was there.

**Comment by bsipocz** [comment]
@lpsinger - That's a relatively new addition, came with the gaia module.

**Comment by bsipocz** [comment]
cc @stargaser

**Comment by vandesai1** [comment]
While it is true that IRSA prefers that users use VO protocols when possible, there are some important data sets that are still only available through IBE. As a representative from IRSA, I would not recommend removing IBE from astroquery at this time. @lpsinger @keflavich

**Comment by lpsinger** [comment]
> While it is true that IRSA prefers that users use VO protocols when possible, there are some important data sets that are still only available through IBE. As a representative from IRSA, I would not recommend removing IBE from astroquery at this time.

The ibe module does not work currently because some of the URLs and table organization have changed. It looks like IBE queries are getting served using HTTP redirects to the TAP interface. @vandesai1, what if we re-implemented the ibe module using `astroquery.utils.tap` under the hood as @bsipocz suggests?

**Comment by vandesai1** [comment]
@lpsinger, yes that should be possible. One difficulty may be preserving the 'mcen' feature that returns the most centered image.

File: astroquery/ibe/__init__.py [hunk 1]
```diff
@@ -3,43 +3,14 @@
 IRSA Image Server program interface (IBE) Query Tool
 ====================================================
 
-This module contains various methods for querying the
-IRSA Image Server program interface (IBE).
+The ``astroquery.ibe`` module is deprecated and will be removed in a future
+release of Astroquery. IRSA no longer supports the IBE interface. To access
+IRSA datasets programatically, users encouraged to use the TAP protocol
+(http://irsa.ipac.caltech.edu/docs/program_interface/TAP.html) via PyVO
+(https://pyvo.readthedocs.io/en/latest/dal/index.html#table-access-protocol).
```
Comment (stargaser) [inline astroquery/ibe/__init__.py]: I am pretty sure the Astropy Project wants to encourage users to use `astropy.utils.tap` and not redirect them to the `pyvo` package. The TAP functionality currently in astroquery works with the main Gaia DR1 archives in Europe but some small changes are needed to make it work with IRSA TAP.

Comment (stargaser) [reply astroquery/ibe/__init__.py]: (oh sorry -- this was addressed in the main conversation already)

**Comment by bsipocz** [comment]
I'm closing this PR now. The IBE/IRSA modules need a rework to use pyvo and the VO API under the hood, but they should not be removed atm.

Also, to wrap up the utils.tap vs pyvo convo: a lot has happened in the past 2 years, and the roadmap right now is to use pyvo for all vo related functionality and remove utils.tap as soon as the modules relying on it made the switch (where the switch depends on getting a few extra functionalities into pyvo first).

**Pull request closed without merging**